uelc/main/tests/test_helper_functions.py

```diff
@@ -0,0 +1,41 @@
+from django.test import TestCase
+# from django.test.client import Client
+from pagetree.helpers import get_hierarchy
+from factories import GroupUpFactory, UELCModuleFactory
+# from pagetree.tests.factories import ModuleFactory
+# from quizblock.tests.test_models import FakeReq
+# from uelc.main.models import CaseQuiz
+from uelc.main.helper_functions import (
+    admin_ajax_page_submit, admin_ajax_reset_page)
+
+
+class TestHelperFunctions(TestCase):
+
+    def setUp(self):
+        UELCModuleFactory("main", "/pages/main/")
+        self.hierarchy = get_hierarchy(name='main')
+        self.section = self.hierarchy.get_root().get_first_leaf()
+        self.grp_usr_profile = GroupUpFactory()
+
+    def test_admin_ajax_page_submit(self):
+        '''Prior to running method it should be false no?'''
```

Comment:
I could be wrong but it will always be unlocked if there are no blocks in the section that need to be submitted. If there is a (need to submit) block like a decision block, then it should return false on first check. So you may need to add a decision block in here...